wallet-new/src/Cardano/Wallet/API/Indices.hs

```diff
@@ -33,6 +35,15 @@ instance ToIndex Wallet Core.Coin where
         Just c  -> Just (Core.mkCoin c)
     accessIx Wallet{..} = let (V1 balance) = walBalance in balance
 
+instance ToIndex Transaction Core.TxId where
+    toIndex _ = rightToMaybe . decodeHash
+    accessIx Transaction{..} = let V1 ti = txId in ti
```

Comment:
Is there a reason we don't have an `unV1 :: V1 a -> a` ?

Reply:
No reason 😛